quartz/plugins/transformers/ofm.ts

```diff
@@ -414,8 +414,8 @@ export const ObsidianFlavoredMarkdown: QuartzTransformerPlugin<Partial<Options>
                 return
               }
 
-              // find first line
-              const firstChild = node.children[0]
+              // find first line and callout content
+              const [firstChild, calloutContent] = node.children
```

Comment:
can you see if this works if you have more than one child node in the blockquote?

```
> [!info] hello
>
> - test
> - two
> 
> hello
>
> - **another test**
```

Reply:
Fixed it.

```html
<blockquote class="callout info" data-callout="info">
  <div class="callout-title">
    <div class="callout-icon"></div>
    <div class="callout-title-inner">
      <p>hello </p>
    </div>
  </div>
  <div class="callout-content">
    <ul>
      <li>test</li>
      <li>two</li>
    </ul>
    <p>hello</p>
    <ul>
      <li><strong>another test</strong></li>
    </ul>
  </div>
</blockquote>
```

I updated the `splice` at the end as well. The `Infinity` passed is apparently the recommended way to do this.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice#deletecount